templates/mathlib_stats.html

```diff
@@ -5,18 +5,18 @@
 
       <h1>Mathlib statistics</h1>
 
-    <h2>Declaration Counts</h2>
+    <h2>Counts</h2>
 
     <table class="table table-borderless table-sm mt-4 ml-5" style="font-size: 1.25rem">
         <tr>
             <th>Definitions</th>
             <th>Theorems</th>
-            <th>Meta declarations</th>
+            <th>Contributors</th>
         </tr>
         <tr>
             <td>{{ num_defns }}</td>
```

Comment:
Possibly
```suggestion
            <td>{{ num_defns }} (+ {{num_meta}} meta)</td>
```
Could link `meta` to https://leanprover-community.github.io/extras/tactic_writing.html, although it's not a perfect match.

Reply:
IMO this is more clutter than it's worth, not sure the meta number is particularly interesting to most people.

Reply:
We could hide more detailed statistics inside a `<details>` dropdown.

Reply:
Probably something for another PR though.